### PR TITLE
`Exam mode`: Fix an issue where notification button and message do not show up during exam

### LINKED ISSUE #8880: `Exam mode`: Notification button and message does not show up during exam
### Describe the bug

Notification button and message does not show up during exam but it shows up in welcome page and end page of the exam. 

### To Reproduce

1. Go to course management as instructor/admin
2. Click on exams
3. Select the exam
4. Edit the exam time 
5. Login with student account
6. Participate exam 
7. See that the notification does not show up during exam, but it shows up in welcome page and end page

### Expected behavior

Notification button and message should be displayed during the exam

### Screenshots

_No response_

### Which version of Artemis are you seeing the problem on?

7.3.1

### What browsers are you seeing the problem on?

Chrome

### PR DESCRIPTION
**Pull request by edkaya**
#### General

- [x] I tested **all** changes and their related features with **all** corresponding user types on a test server.
- [ ] This is a small issue that I tested locally and was confirmed by another developer on a test server.
- [x] Language: I followed the [guidelines for inclusive, diversity-sensitive, and appreciative language](https://docs.artemis.cit.tum.de/dev/guidelines/language-guidelines/).
- [x] I chose a title conforming to the [naming conventions for pull requests](https://docs.artemis.cit.tum.de/dev/development-process/development-process.html#naming-conventions-for-github-pull-requests).

#### Client
- [x] I **strictly** followed the [client coding and design guidelines](https://docs.artemis.cit.tum.de/dev/guidelines/client/).
- [x] Following the [theming guidelines](https://docs.artemis.cit.tum.de/dev/guidelines/client-design/), I specified colors only in the theming variable files and checked that the changes look consistent in both the light and the dark theme.
- [x] I added multiple screenshots/screencasts of my UI changes.

### Motivation and Context

In the redesigned student view of exam mode, notification button and message do not show up during the exam, but they show up in welcome and end page of the exam. Therefore, I added this button in the exam bar. It fixes #8880.

### Description

The notification button is added into exam-bar to be displayed during the exam. Besides, 'Notification' lettering is also removed, since the icon of the button itself describes its purpose anyways.

### Steps for Testing

Prerequisites:
- 1 Admin
- 1 Student
- 1 Exam

1. Log in to Artemis
2. Participate the exam
3. Verify that there is no notification button
4. Log out and log in with admin account
5. Edit the time of the exam 
6. Log in with student acount again 
7. Participate exam
8. Verify that there is notification button between hand in early button and remaining time 
9. Verify that the popup information is correct after clicking on this button

#### Exam Mode Testing

Prerequisites:
- 1 Student
- 1 Exam 
- 
1. Log in to Artemis
2. Participate in the exam as a student
3. Make sure that the UI of other components in the exam mode stays unchanged. You can use the [exam mode documentation](https://docs.artemis.cit.tum.de/user/exams/students_guide.html#participating-in-the-artemis-online-exam) as reference.

[](https://artemis-test1.artemis.cit.tum.de)
[](https://artemis-test2.artemis.cit.tum.de)
[](https://artemis-test3.artemis.cit.tum.de)
[](https://artemis-test4.artemis.cit.tum.de)
[](https://artemis-test5.artemis.cit.tum.de)
[](https://artemis-test6.artemis.cit.tum.de)
[](https://artemis-test9.artemis.cit.tum.de)

#### Code Review
- [x] Code Review 1
- [x] Code Review 2
#### Manual Tests
- [x] Test 1
- [x] Test 2
#### Exam Mode Test
- [x] Test 1
- [x] Test 2

### Screenshots

Screenshot 2024-06-28 at 17 12 51

Screenshot 2024-06-28 at 17 12 59

## Summary by CodeRabbit

- **New Features**
  - Added a live events button to the exam interface, displaying the count of live events.

- **Enhancements**
  - Simplified the display text of the live events button to show only the count of events, improving clarity and user experience.
  - Restructured the exam bar layout to include the live events button and other elements more effectively.

### CODE REVIEW

**Review by pzdr7 — DISMISSED** [review]
Tested on TS5 with

- changing the problem statement of a text / programming exercise ✅ 
- changing the total working time ✅ 
- changing the individual working time ✅ 

Works as expected 👍

**Review by SimonEntholzer — APPROVED** [review]
Code looks good and tested on Ts1

**Review by JohannesStoehr — APPROVED** [review]
Code

**Review by sarpsahinalp — DISMISSED** [review]
Code lgtm

**Review by egekurt123 — APPROVED** [review]
Code LGTM

File: src/main/webapp/app/exam/participate/events/exam-live-events-button.component.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 @if (eventCount) {
     <button class="btn btn-md btn-info" (click)="openDialog($event)">
         <fa-icon [icon]="faBullhorn" />
-        <span class="d-none d-md-inline">{{ 'artemisApp.exam.events.button' | artemisTranslate }} ({{ eventCount }})</span>
+        <span class="d-none d-md-inline">({{ eventCount }})</span>
```
Comment (rabeatwork) [inline src/main/webapp/app/exam/participate/events/exam-live-events-button.component.html:4]: Currently, the event count disappears on small screens, right? I would like to display it on small screens as well. 
That means simply removing the two classes.

**Review by egekurt123 — APPROVED** [review]
Reapprove

**Review by krusche — APPROVED** [review]
Changes look good to me 👍

**Pull request merged**